backend/src/config.py

```diff
@@ -8,8 +8,9 @@ class Settings(BaseSettings):
     secret_key: str
     home_location: str
     mongodb_uri: str
+    csrf_secret_key: str
 
-    model_config = SettingsConfigDict(env_file="../.env", extra="ignore")
+    model_config = SettingsConfigDict(env_file=".env", extra="ignore")
```

Comment:
```suggestion
    model_config = SettingsConfigDict(env_file="../.env", extra="ignore")
```
Currently setup doesn't differentiate between `.env` file for frontend and backend (in their own directories), but expects just one in the root folder.

Related to this - could you add to the `sample.env` new required variable with some placeholder?

Reply:
Hey @willhitman,

this wasn't addressed yet, but it was marked as resolved. Could you take another look?